<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2002</width>
    <height>1176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>1301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../ImagenesGUI/title.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>710</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Birth">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SinGenerator">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="CantidadHumans">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="GAGenerator">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SinMenu">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="GAMenu">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Salvation">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>140</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Damnation">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>220</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>821</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ABB">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>740</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Hell">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>410</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Heaven">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>300</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Maps">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>530</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Maps2">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>630</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2002</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
